code/modules/antagonists/nukeop/outfits.dm

```diff
@@ -52,8 +52,8 @@
 	if(command_radio)
 		radio.command = TRUE
 		radio.use_command = TRUE
-
-	if(ispath(uplink_type, /obj/item/uplink/nuclear) || tc) // /obj/item/uplink/nuclear understands 0 tc
+	// Renamed from /obj/item/uplink/nuclear so it works with other subtypes (ie. clown op reinforcement)
```

Comment:
```suggestion
```

this comment is historical and no longer necessary for people to understand this code.

Reply:
i'll fix it for you

Reply:
Thanks